backend/spellchecker/server/spellchecker.py

```diff
@@ -14,7 +15,7 @@ class Spellchecker:
     """
     def __init__(self, path: str = '', language: str ='', log: object = None):
         self._hunspell_instances = {}
-        self._parser = Parser()
+        self._parser = Parser("../libparser/build/libparser.so")
```

Comment:
Come on. The point was not in the moving string literal from one place in the code into another. The point is to make it configurable externally, via command-line arguments. The same applies to port number.

Hint: argparse package.